tox.ini

```diff
@@ -20,7 +20,7 @@ commands =
 
 [testenv:flake8]
 deps = flake8
-commands = flake8
+commands = flake8 --exclude=migrations rest_framework_json_api example
```

Comment:
flake8 configuration is to be found in the `setup.cfg`. I think for consistency we should move all options there.

Reply:
I'll clean up my mess. Just found another small error in the deprecation warnings as well.